dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -22,9 +22,10 @@ arrow-memory-netty/2.0.0//arrow-memory-netty-2.0.0.jar
 arrow-vector/2.0.0//arrow-vector-2.0.0.jar
 audience-annotations/0.5.0//audience-annotations-0.5.0.jar
 automaton/1.11-8//automaton-1.11-8.jar
-avro-ipc/1.8.2//avro-ipc-1.8.2.jar
-avro-mapred/1.8.2/hadoop2/avro-mapred-1.8.2-hadoop2.jar
-avro/1.8.2//avro-1.8.2.jar
+avro-ipc-jetty/1.10.1//avro-ipc-jetty-1.10.1.jar
```

Comment:
Do we need `avro-ipc-jetty-1.10.1`?

Reply:
This change was auto-generated because the transitive dependencies of avro-mapred changed and now include avro-ipc-jetty. Spark probably does not need this at all, but I let it because I didn't want to change by hand the results of the script.

Reply:
If then, shall we try to add `exclusion` at `pom.xml`?

Reply:
ok doing it